replybot/lib/typewheels/machine.js

```diff
@@ -522,6 +522,13 @@ function _gatherResponses(ctx, qa, q, previous = []) {
 
 function _response (ctx, qa, {question, validation, response, token, followUp}) {
 
+  if (ctx.surveyMetadata && ctx.surveyMetadata.off_date < ctx.timestamp) {
```

Comment:
Hrm, so even though the table in the database is called "surveys", everywhere here in replybot it's referred to as a "form"... So I would definitely try and keep it consist it within an app.

Reply:
Wait, no, nevermind, the metadata applies to the whole survey, not just the form. I take it back!

Reply:
Why is it in reference to ctx.timestamp and not response.timestamp?